src/server/index.jsx

```diff
@@ -292,6 +295,7 @@ server.get(
           requestUrl: url,
         });
 
+        console.error(error);
```

Comment:
Will this have any side effects when running on our EC2's if this spits out errors to the console?

Reply:
Yes, good shout!